### PR TITLE
Blog backend

### PR DESCRIPTION
**Pull request by magsyg**
Added blog to backend
added the general tests for it aswell

### CODE REVIEW

File: backend/samfundet/conftest.py [hunk 1]
```diff
@@ -170,3 +170,10 @@ def fixture_informationpage() -> Iterator[InformationPage]:
     informationpage = InformationPage.objects.create(title_nb='Norsk tittel', title_en='Engel', slug_field='Sygard')
     yield informationpage
     informationpage.delete()
+
+
+@pytest.fixture
+def fixture_blogpost(fixture_image: Image) -> Iterator[BlogPost]:
+    blogpost = BlogPost.objects.create(title_nb='Tittel', title_en='Title', text_nb='halla verden', text_en='hellow world uWu', image=fixture_image)
```
Comment (emilte) [inline backend/samfundet/conftest.py:177]: trailing comma readability

File: backend/samfundet/tests/test_views.py
Comment (emilte) [inline backend/samfundet/tests/test_views.py:1]: Fantastisk! Takk Magnus

**Review by emilte — APPROVED** [review]
Nice. Nesten morsomt hvor boilerplate det har blitt

File: backend/samfundet/conftest.py (hunk 1 above)
Comment (magsyg) [reply backend/samfundet/conftest.py]: Done

File: backend/samfundet/tests/test_views.py
Comment (magsyg) [reply backend/samfundet/tests/test_views.py:1]: Takk! :D

**Pull request merged**